main/mafia.hs

```diff
@@ -500,14 +502,39 @@ ghciArgs extraIncludes paths = do
 
   extras <- concat <$> mapM reifyInclude extraIncludes
 
-  let dirs = ["src", "test", "gen", "dist/build/autogen"] <> extras
-  includes  <- catMaybes <$> mapM ensureDirectory dirs
+  let
+    dirs =
+      ["src", "test", "gen", "dist/build/autogen"] <> extras
```

Comment:
oh, right no I meant do I need to include all the sub-directories of `dist/build` recursively, but no, only `dist/build` and `dist/build/autogen`, cool